ValidationLibrary.AzureFunctions/Startup.cs

```diff
@@ -48,6 +48,19 @@ public void Configure(IWebJobsBuilder builder)
                 .AddEnvironmentVariables()
                 .Build();
 
+            // Get all rule classes.
+            var assembly = Assembly.Load("ValidationLibrary.Rules, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null");
+            var validationRules = assembly.GetExportedTypes().Where(c => c.GetInterfaces().Contains(typeof(IValidationRule)));
```

Comment:
Where does `c` come from here?

Reply:
Well I think it was c for class but I changed it to t for type. It's for the LINQ method.